app/services/workers/send-accepted-claim-notification.js

```diff
@@ -1,12 +1,98 @@
 const config = require('../../../config')
 const sendNotification = require('../notify/send-notification')
+const getApprovedClaimExpenseData = require('../data/get-approved-claim-expense-data')
 
 module.exports.execute = function (task) {
+  var claimId = task.claimId
   var reference = task.reference
   var personalisation = {reference: reference} // TODO payment breakdown
+  var claimExpenses
 
   var emailAddress = task.additionalData
   var emailTemplateId = config.NOTIFY_ACCEPTED_CLAIM_EMAIL_TEMPLATE_ID
+  return getApprovedClaimExpenseData.getClaimExpenseData(claimId)
+    .then(function(claimExpenseData) {
+      claimExpenses = claimExpenseData
+    })
+    .then(function() {
+      return getApprovedClaimExpenseData.getClaimantData(reference, claimId)
```

Comment:
as above, change to single function which returns {claimExpenseData: claimExpenseData, claimantData: claimantData}